xml/2018-07-25-barbecue.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<email title="DoCSoc Summer BBQ" agenda="false">
+
+  <text>
+    Hi everyone,
+
+    Hope your summer holidays are going well! With the great weather, DoCSoc
+    will be holding a summer barbecue in a couple weeks' time. If you're around
+    then, take a look at the event details below!
+  </text>
+
+  <hr/>
+
+  <event startDate="2018-08-12T13:00:00+01:00"
```

Comment:
heheee sorry about the ISO 8601

Reply:
To be honest, I actually prefer ISO 8601 :P